src/GraphQL/Execution/UnhandledExceptionContext.cs

```diff
@@ -10,7 +10,7 @@ public class UnhandledExceptionContext
         /// <summary>
         /// Initializes a new instance with the specified properties.
         /// </summary>
-        public UnhandledExceptionContext(ExecutionContext context, IResolveFieldContext fieldContext, Exception originalException)
+        public UnhandledExceptionContext(IExecutionContext context, IResolveFieldContext fieldContext, Exception originalException)
```

Comment:
It seemed that in this situation, a read-only view of the execution context would be appropriate.